services/searchengine/bleveengine/bleve_test.go

```diff
@@ -138,7 +138,7 @@ func (s *BleveEngineTestSuite) TestDeleteChannelPosts() {
 		userID := model.NewId()
 		channelID := model.NewId()
 		channelToDeleteID := model.NewId()
-		post := createPost(userID, channelID, "test one two three")
+		post := createPost(userID, channelID, model.NewRandomString(15))
```

Comment:
I think here (and in the other related cases) the best approach would be to remove the parameter and generate the random message directly inside `createPost`.

Reply:
@mahmud2011 - can you please look into this?

Reply:
@agnivade Sorry, I missed it. Happy to do that.